helpers/providers/index.ts

```diff
@@ -29,14 +30,12 @@ export async function askModelConfig({
         name: "provider",
         message: "Which model provider would you like to use",
         choices: [
-          {
-            title: "OpenAI",
-            value: "openai",
-          },
+          { title: "OpenAI", value: "openai" },
           { title: "Groq", value: "groq" },
           { title: "Ollama", value: "ollama" },
           { title: "Anthropic", value: "anthropic" },
           { title: "Gemini", value: "gemini" },
+          { title: "T-Systems", value: "t-systems" },
```

Comment:
we're running here in a bug if the user selects another framework but FastAPI.

Options:
1. add a T-Systems template also for Typescript (preferable)
2. pass the `framework` variable to `askModelConfig` and only show T-Systems for fastapi

Reply:
We can go with option 2 first as option 1 would take some time to implement.